### PR TITLE
Formvalidation

### PR DESCRIPTION
**Pull request by safstromo**

### CODE REVIEW

**Comment by safstromo** [comment]
[visa felmeddelande vid fel inloggning](https://trello.com/c/8cobK6NI/152-visa-felmeddelande-vid-fel-inloggning)

**Comment by safstromo** [comment]
[Validera signup form](https://trello.com/c/5dK8WBYo/156-validera-signup-form)

File: src/components/LoginModal.vue [hunk 1]
```diff
@@ -49,43 +48,50 @@ async function login() {
...
-        <label class="text-whiteBackRow" for="password"><b>Password</b></label>
-        <input
+        <FormKit
+          label="Password"
           type="password"
-          v-model="data.password"
+          validation="required|length:6|matches:/[^a-zA-Z]/"
+          :validation-messages="{
+            matches: 'Please include at least one symbol'
+          }"
```
Comment (figgge) [inline src/components/LoginModal.vue:61]: ``
```suggestion

``

Do we need validation when logging in?

**Review by figgge — CHANGES_REQUESTED** [review]

**Review by figgge — APPROVED** [review]
Great work!

**Pull request merged**